platforms/documentation/docs/src/docs/userguide/reference/runtime-configuration/gradle_wrapper.adoc

```diff
@@ -79,11 +80,19 @@ The generated Wrapper properties file, `gradle/wrapper/gradle-wrapper.properties
 
 The following is an example of the generated distribution URL in `gradle/wrapper/gradle-wrapper.properties`:
 
-[source,text]
+[source,properties,subs="attributes"]
 ----
 distributionUrl=https\://services.gradle.org/distributions/gradle-{gradleVersion}-bin.zip
 ----
 
+[TIP]
+====
+Use the `-bin` distribution for most builds.
+
+The `-bin` distribution contains only the runtime needed to build and run Gradle.
+It’s smaller to download and faster to cache on CI systems compared to the `-all` distribution, which also includes Gradle’s full source code and documentation.
```

Comment:
💅  I'd add something about "modern IDEs can download Gradle sources automatically when needed, so you don't have to think about it".